src/service/dynamo.ts

```diff
@@ -237,13 +257,22 @@ class Dynamo implements DynamoService {
           dependencies: true,
         },
       }),
-    });
+    };
+
+    let response = await this._fetch(`${this.url}/v1/graph/info`, reqData);
+    const maxRetries = 3;
+    let ii = 0;
+    while (response.status === 500 && ii <= maxRetries) {
+      response = await this._fetch(`${this.url}/v1/graph/info`, reqData);
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
if await `this._fetch` throws in case of 500, then we need to wrap it in a try catch